discord/commands.py

```diff
@@ -42,45 +39,51 @@ def __new__(cls, *args, **kwargs):
 
         self.__original_kwargs__ = kwargs.copy()
         return self
+
     def __init__(self, func, *args, **kwargs):
         if not asyncio.iscoroutinefunction(func):
-            raise TypeError('Callback must be a coroutine.')
+            raise TypeError("Callback must be a coroutine.")
+        self.callback = func
 
-        name = kwargs.get('name') or func.__name__
+        self.guild_ids = kwargs.get("guild_ids", None)
+
+        name = kwargs.get("name") or func.__name__
         if not isinstance(name, str):
-            raise TypeError('Name of a command must be a string.')
+            raise TypeError("Name of a command must be a string.")
         self.name = name
 
-        description = kwargs.get('description') or inspect.cleandoc(func.__doc__)
-        if description == None:
-            description = "No description set"
-        elif not isinstance(name, str):
-            raise TypeError('Description of a command must be a string.')
+        description = (
+            kwargs.get("description")
+            or inspect.cleandoc(func.__doc__)
+            or "No description set"
```

Comment:
We should raise an error if no description is given. it makes more sense as an extension of the API